src/kaltura-player.js

```diff
@@ -584,13 +608,17 @@ class KalturaPlayer extends FakeEventTarget {
       Playlist: PlaylistEventType,
       UI: UIEventType,
       // For backward compatibility
-      ...CoreEventType
+      ...CoreEventType,
+      ...{VISIBILITY_CHANGE: VISIBILITY_CHANGE}
```

Comment:
```suggestion
      VISIBILITY_CHANGE
```

Reply:
I don't how magic works :)

Reply:
Amazing, learnt something new. Thanks!

Reply:
@RoyBregman https://alligator.io/js/object-property-shorthand-es6/

Reply:
yeah, I already found it in some site. Really cool!